utils/utils.go

```diff
@@ -176,3 +177,44 @@ func RandomString(strlen int) string {
 	}
 	return string(result)
 }
+
+// RemoveDuplicates returns the slice removing duplicates if exist
+func RemoveDuplicates(elements []string) []string {
+	encountered := map[string]bool{}
+
+	// Create a map of all unique elements.
+	for v := range elements {
+		encountered[elements[v]] = true
+	}
+
+	// Place all keys from the map into a slice.
+	result := []string{}
+	for key := range encountered {
+		result = append(result, key)
+	}
+	return result
+}
+
+// Contains evaluates whether s contains x.
+func Contains(s []string, x string) bool {
+	for _, n := range s {
+		if x == n {
+			return true
+		}
+	}
+	return false
+}
+
+// Subset returns true if the first slice is completely contained in the second slice.
+// There must be at least the same number of duplicate values in second as there are in first.
+func Subset(s1, s2 []string) bool {
```

Comment:
I think this method is not used anywhere in the codebase

Reply:
yes, there are a lot of them... My idea is to remove all of them in close future, refactoring and normalizing focused in these aspects